src/resources/triton_python_backend_utils.py

```diff
@@ -295,6 +296,184 @@ def triton_string_to_numpy(triton_type_string):
     return TRITON_STRING_TO_NUMPY[triton_type_string]
 
 
+class ModelConfig:
+    """An object of ModelConfig class is used to describe
+    the model configuration for autocomplete.
+    Parameters
+    ----------
+    model_config : ModelConfig Object
+        Object containing the model configuration. Only the max_batch_size, inputs
+        and outputs properties can be modified for auto-complete model configuration.
+    """
+
+    def __init__(self, model_config):
+        self._model_config = json.loads(model_config)
+
+    def __str__(self):
+        return str(self._model_config)
+
+    def as_dict(self):
+        """Provide the read-only access to the model configuration
+        Returns
+        -------
+        dict
+            dictionary type of the model configuration contained in
+            the ModelConfig object
+        """
+        return self._model_config
+
+    def set_max_batch_size(self, max_batch_size):
+        """Set the max batch size for the model.
+        Parameters
+        ----------
+        max_batch_size : int
+            The max_batch_size to be set.
+        Raises
+        ------
+        ValueError
+            If configuration has specified max_batch_size non-zero value which
+            is larger than the max_batch_size to be set for the model.
+        """
+        if self._model_config["max_batch_size"] > max_batch_size:
+            raise ValueError(
```

Comment:
Should reformat the error message similar to others which helps users localize the error to auto-complete-config implementation.  Also include the model name.

Reply:
Sorry I missed modifying this error message. Updated, please review.